python/cudf_polars/tests/experimental/test_groupby.py

```diff
@@ -62,14 +62,10 @@ def test_groupby_single_partitions(df, op, keys):
     )
 
 
-@pytest.mark.parametrize(
-    "op", ["sum", "mean", "len", "count", "min", "max", "n_unique"]
-)
-@pytest.mark.parametrize("keys", [("y",), ("y", "z")])
+@pytest.mark.parametrize("op", ["n_unique"])
```

Comment:
I might be misreading the diff, but are we losing test coverage for all the cases we've removed (the other operations, and the multi-column grouper)? Or is this already covered elsewhere?